readme.txt

```diff
@@ -0,0 +1,44 @@
+=== User Interface Options ===
+Contributors: Fluid Project
+Tags: accessibility, personalization
+Requires at least: 4.1.1
+Tested up to: 4.1.1
+License: New BSD license or the Educational Community License, Version 2.0
+License URI: https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+
+This plugin adds the Infusion User Interface Options component to the site.
+
+== Description ==
+
+This plugin adds the Infusion User Interface Options (UIO) component to the site. UIO creates a
+sliding panel at the top of the page that contains controls which allow visitors to customize the
+presentation of the site to their own personal needs and preferences.
+
+For more information, contact the [Fluid-work mailing list](mailto:fluid-work@fluidproject.org)
+or join the [fluid-work IRC chat room](http://wiki.fluidproject.org/display/fluid/IRC+Channel).
+
+== Installation ==
+
+1. Upload `user-interface-options.zip` to the `/wp-content/plugins/` directory and unpack it.
```

Comment:
This is actually markdown, so the rendering will figure it out, but for readability, you're probably right. I'll update it.